src/handleLoginCliCommand.ts

```diff
@@ -39,7 +39,7 @@ export const handleLoginCliCommand = async (
 
 	const tokenTxtPath = join(homedir(), '.intuita', 'token.txt');
 
-	await writeFile(tokenTxtPath, token);
+	await writeFile(tokenTxtPath, token, 'utf-8');
```

Comment:
why was it needed? or did you prefer to do it just to be safe?

Reply:
just to keep the consistency
not really needed